test/controllers/posts_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /index' do
+    before(:example) { get '/users/sample_user' }
+    it 'Should have http status ok' do
+      expect(response).to have_http_status(:ok)
+    end
+    it 'should render posts index page for a user' do
+      expect(response.body).to include('User: sample_user')
+    end
```

Comment:
- [ ] For this, ensure the response body includes the correct placeholder text. please target the correct placeholder that is appearing on the body.

Reply:
Hello @Emmyn5600,
Thank you for your review. I didn't quite understand what you meant by these comments
Here I am checking for a text `'User: sample_user'` when I provide an id `sample_user`
Could you please elaborate on the changes I have to make

Reply:
@eapenzacharias what I meant was to target the correct placeholder text that is appearing on the body to ensure that we are getting the correct output after targeting that text.

Reply:
Hello @Emmyn5600

Thank you for the clarification. I believe in these three cases I am targeting the right placeholders. Could you please verify and check if I am missing.